appsec-kit-demo-v8/src/main/java/com/vaadin/appsec/demo/v8/DemoUI.java

```diff
@@ -11,18 +11,11 @@
 
 import javax.servlet.annotation.WebServlet;
 
-import org.cyclonedx.model.Component;
-
 import com.vaadin.annotations.Push;
 import com.vaadin.annotations.Theme;
 import com.vaadin.annotations.VaadinServletConfiguration;
-import com.vaadin.appsec.backend.model.osv.response.OpenSourceVulnerability;
-import com.vaadin.appsec.backend.service.BillOfMaterialsStore;
-import com.vaadin.appsec.backend.service.VulnerabilityStore;
-import com.vaadin.appsec.demo.util.OpenSourceVulnerabilityUtils;
```

Comment:
We could remove the `OpenSourceVulnerabilityUtils` class as it is not used anymore in the demo application.